modules/core/src/test/java/org/apache/ignite/testframework/junits/common/KeyType.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.ignite.testframework.junits.common;
+
+/**
+ * Defined type of key
```

Comment:
Please, format java doc properly.
Sentence should end with the dot.

Reply:
Ok.